src/SharedResources/components/Label.ts

```diff
@@ -0,0 +1,24 @@
+import { FunctionComponent, createElement } from "react";
+import * as classNames from "classnames";
+
+export interface LabelProps {
+    className?: string;
+    label: string;
+    weight: number;
+    style?: object;
+    orientation?: "horizontal" | "vertical";
+}
+
+export const Label: FunctionComponent<LabelProps> = ({ children, className, label, style, weight, orientation }) => {
+    weight = (weight > 11 || weight < 1) ? 3 : weight;
+    const labelWeight = orientation === "horizontal" ? `col-sm-${weight}` : "col-sm-1";
```

Comment:
col-sm-1 will not work with huge label contents, because it will fill just 1/12 of the space